crates/sui-snapshot/src/reader.rs

```diff
@@ -219,8 +219,9 @@ impl StateSnapshotReaderV1 {
         }
         checksum_progress_bar.finish_with_message("Checksumming complete");
 
-        // Spawn accumulation task
-        info!("Accumulating ref files for verification");
+        // Spawn accumulation progress bar
```

Comment:
nit: Could we possibly refactor and move the whole accumulator code into a separate function which returns the join handle? Suggesting because this function has gotten too big

Reply:
Done